sidecar/tests/test_gpt_sovits_g2p.py

```
@@ -0,0 +1,126 @@
+    import pickle
+    for rel in ("G2P/EnglishG2P/engdict_cache.pickle",
+                "G2P/EnglishG2P/namedict_cache.pickle",
+                "G2P/ChineseG2P/polyphonic.pickle"):
+        src = os.path.join(ASSETS, rel)
+        dst = src[: -len(".pickle")] + ".json"
+        if os.path.isfile(src) and not os.path.isfile(dst):
+            with open(src, "rb") as f:
+                # Safe by scope: this is a dev-only fixture path the developer
+                # opts into via SOKUJI_GPT_SOVITS_ASSETS (upstream GenieData
+                # they fetched themselves); production loaders never unpickle.
+                data = pickle.load(f)
```

Ruff (0.15.21)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)